pkg/ui/react-app/src/thanos/pages/blocks/helpers.ts

```diff
@@ -54,3 +54,9 @@ export const sortBlocks = (blocks: Block[], label: string): { [source: string]:
   });
   return sortedPool;
 };
+
+export const download = (blob: Block): string => {
+  const url = window.URL.createObjectURL(new Blob([JSON.stringify(blob)], { type: 'application/json' }));
```

Comment:
You can do `JSON.stringify(blob, null, 2)` to get properly indented json as output.

Reply:
Done this and it works fine